koku/api/report/queries.py

```diff
@@ -174,6 +174,25 @@ def list_months(start_date, end_date):
             current = next_month
         return months
 
+    @staticmethod
+    def lists_to_set(list_a, list_b):
+        """Merge two lists into a set.
+
+        Args:
+            list_a (List[String]): List of strings
+            list_b (List[String]): List of strings
+        Return:
+            (Set): unique set of strings
+        """
+        out = set()
+        if list_a:
```

Comment:
`list_a` and `list_b` are both required arguments. So, checking the truthiness here isn't necessary.

This entire function can be simplified down to `return set(list_a + list_b)`

Reply:
These values can be `None` and `set(None + None)` doesn't work.

Reply:
Changed it so that these values are empty lists not None now and removed lists_to_set